integration-tests/src/test_loop/utils/trie_sanity.rs

```diff
@@ -350,17 +355,35 @@ pub fn check_state_shard_uid_mapping_after_resharding(
         epoch_config.shard_layout.get_children_shards_uids(parent_shard_uid.shard_id()).unwrap();
```

Comment:
nit though not your code:
It's more typical to just get the shard layout directly from the epoch manager instead of going through the epoch config.